src/apps/explorer/components/common/BlockExplorerLink/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+import { DumbBlockExplorerLink, Props } from 'components/common/DumbBlockExplorerLink'
```

Comment:
I don't think this is App specific

Any app using the same `state/network` could use this smart component.